providers/implementations/kem/rsa_kem.c

```diff
@@ -270,6 +284,9 @@ static int rsasve_generate(PROV_RSA_CTX *prsactx,
     int ret;
     size_t nlen;
 
+    if (!ossl_prov_is_running())
```

Comment:
This check should be done in `rsakem_generate()` I think.
That's the only caller of this function and it matches the check added in `rsasve_recover()`.